code/sound/sound.cpp

```diff
@@ -300,14 +300,15 @@ static std::unique_ptr<sound::IAudioFile> openAudioFile(const char* fileName)
 // a single instance, and can be played multiple times simultaneously.  Through the magic of
 // DirectSound, only 1 copy of the sound is used.
 //
-// parameters:		gs							=> file of sound to load
+// parameters:		entry							=> entry of sound to load
+// parameters:		flags							=> flags of sound to load
```

Comment:
Not a blocker, but could you update this comment to "pointer to flags of sound to load (can be null)" or similar?

Reply:
Ya